jupyterlab_hdf/tests/test_meta.py

```diff
@@ -105,3 +108,11 @@ def test_empty_dataset(self):
         assert response.status_code == 200
         payload = response.json()
         assert payload == dict((("attributes", []), ("dtype", ">f8"), ("labels", None), ("name", "empty"), ("ndim", 0), ("shape", None), ("size", 0), ("type", "dataset")))
+
+    def test_external_link(self):
+        response = self.tester.get(["meta", "test_file.h5"], params={"uri": "/external"})
+
+        assert response.status_code == 200
+        payload = response.json()
+
+        assert payload == dict((("name", "external"), ("targetFile", "another_file.h5"), ("targetUri", "path/in/the/file"), ("type", "externalLink")))
```

Comment:
Note that these are only additions: the resulting endpoint responses are unchanged.